cli/azd/pkg/input/console.go

```diff
@@ -453,6 +481,8 @@ func promptFromOptions(options ConsoleOptions) survey.Prompt {
 	}
 }
 
+const c_afterIO = "oi\n"
```

Comment:
Similar feedback to the above, slight preference for this to be `cAfterIO` as we might do elsewhere.  Here and elsewhere.

Reply:
Just wondering - what does `oi` mean in this context?  I'm sort of confused on how this ends up working.  I feel like the `o` part is always ignored because `updateLastBytes` only considers the last two bytes of the string.  I'm not what (if any) semantics there are for the `i` part.  Could I change it to any other letter?

Reply:
I've added a comment. 
Yes, the letters and length (if greater than 2) don't matter.  We just need a sentinel to set the state after some interactions where we use 3rd party libraries for IO (input/output)